core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -1608,64 +1608,13 @@ trait EntityComponent {
       Entity(entityRecord.name, entityRecord.entityType, attributes)
 
     def unmarshalEntities(
-      entityAttributeRecords: Seq[EntityAndAttributesResult],
-      validateOrdering: Boolean = false
+      entityAttributeRecords: Seq[EntityAndAttributesResult]
     ): Seq[Entity] =
-      unmarshalEntitiesWithIds(entityAttributeRecords, validateOrdering).map { case (_, entity) => entity }
```

Comment:
this removes the `validateOrdering` argument. The last caller that specified that argument was removed in #2665.